parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -232,4 +215,56 @@ cleanUpGPUDrivers() {
     rm -f /etc/apt/sources.list.d/nvidia-docker.list
 }
 
+blacklistNouveau() {
+    tee /etc/modprobe.d/blacklist-nouveau.conf > /dev/null <<EOF
+blacklist nouveau
+options nouveau modeset=0
+EOF
+    retrycmd_if_failure_no_stats 120 5 25 update-initramfs -u || exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
+}
+
+addNvidiaAptRepo() {
```

Comment:
TODO: we can avoid doing this at runtime by checking for the cached versions of the files before invoking the install commands.

probably won't do it in this pr tho